### DISCUSSION TITLE
Top Solutions = Scams and Scoring needs improvment

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#130] [Ideas]
The scoring algorithm needs to be revalued and rethinked,
how people supposed to take that seriously?
that doesn't even display properly ...
![image](https://user-images.githubusercontent.com/58576132/136929322-78bcf01d-2e43-4d26-828b-24a8a258074f.png)
I don't get it how such a sloppy code gets more score than a proper done CSS

**Comment by gunnarbittersmann** [comment]
The scoring algorithm uses a different default font than your local machine does, hence the difference in rendering characters. When using characters, you might want to try Alex Zaworski’s [previewer](https://codepen.io/alexzaworski/full/vwmNKr).
Before anyone would ask: The scoring algorithm has been changed some months ago. The top solution to [target #62](https://cssbattle.dev/play/62) had worked with the old one, but doesn’t anymore. Maybe you’ll find a new top solution that does work?

**Reply by rafehimself** [reply]
I'm trying ... , but no matter what I do, Im 30-50 characters behind the Top,
you gotta learn me some magic tricks ;-) 🤣
